src/main/java/com/google/api/codegen/transformer/StandardSampleImportTransformer.java

```diff
@@ -15,13 +15,17 @@
 package com.google.api.codegen.transformer;
 
 import com.google.api.codegen.config.MethodContext;
+import com.google.api.codegen.config.OutputContext;
 import com.google.api.codegen.metacode.InitCodeNode;
 import com.google.api.codegen.viewmodel.CallingForm;
 import com.google.api.codegen.viewmodel.ImportSectionView;
-import com.google.api.codegen.viewmodel.OutputView;
 import java.util.Collections;
-import java.util.List;
 
+/**
+ * An implementation of SampleImportTransformer. Subclasses of this class can choose to override
+ * `addSampleBodyImports`, `addOutputImports` and `addInitCodeImports` to save language-specific
+ * types to the type table. Currently used by Java.
```

Comment:
Does this transform only the `MethodContext`, or also the `OutputContext`? (Looking at the other files, I gather it's the latter.) Might be helpful to mention that here.

Reply:
Updated this comment and the comment of `addOutputImports`.